[2018-06-19 13:45:53,596] {cli.py:374} INFO - Running on host PRINHYLTPDL1167
[2018-06-19 13:45:53,700] {models.py:1197} INFO - Dependencies all met for <TaskInstance: fileTask.task_1 2018-06-17 08:15:00 [queued]>
[2018-06-19 13:45:53,710] {models.py:1197} INFO - Dependencies all met for <TaskInstance: fileTask.task_1 2018-06-17 08:15:00 [queued]>
[2018-06-19 13:45:53,711] {models.py:1407} INFO - 
--------------------------------------------------------------------------------
Starting attempt 1 of 2
--------------------------------------------------------------------------------

[2018-06-19 13:45:53,745] {models.py:1428} INFO - Executing <Task(BashOperator): task_1> on 2018-06-17 08:15:00
[2018-06-19 13:45:53,746] {base_task_runner.py:115} INFO - Running: ['bash', '-c', u'airflow run fileTask task_1 2018-06-17T08:15:00 --job_id 3 --raw -sd DAGS_FOLDER/fileTask.py']
[2018-06-19 13:45:53,993] {base_task_runner.py:98} INFO - Subtask: /usr/local/lib/python2.7/dist-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
[2018-06-19 13:45:53,994] {base_task_runner.py:98} INFO - Subtask:   """)
[2018-06-19 13:45:54,029] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:45:54,028] {driver.py:120} INFO - Generating grammar tables from /usr/lib/python2.7/lib2to3/Grammar.txt
[2018-06-19 13:45:54,050] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:45:54,050] {driver.py:120} INFO - Generating grammar tables from /usr/lib/python2.7/lib2to3/PatternGrammar.txt
[2018-06-19 13:45:54,166] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:45:54,166] {__init__.py:45} INFO - Using executor SequentialExecutor
[2018-06-19 13:45:54,206] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:45:54,206] {models.py:189} INFO - Filling up the DagBag from /home/charanjiths/airflow/dags/fileTask.py
[2018-06-19 13:45:54,309] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:45:54,309] {bash_operator.py:70} INFO - Tmp dir root location: 
[2018-06-19 13:45:54,309] {base_task_runner.py:98} INFO - Subtask:  /tmp
[2018-06-19 13:45:54,310] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:45:54,310] {bash_operator.py:80} INFO - Temporary script location: /tmp/airflowtmpQDP8ct//tmp/airflowtmpQDP8ct/task_1g5saho
[2018-06-19 13:45:54,310] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:45:54,310] {bash_operator.py:88} INFO - Running command: aws s3 cp Test_File_2018_06_19.txt /home/charanjiths/airflow/ 
[2018-06-19 13:45:54,313] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:45:54,313] {bash_operator.py:97} INFO - Output:
[2018-06-19 13:45:54,315] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:45:54,315] {bash_operator.py:101} INFO - /tmp/airflowtmpQDP8ct/task_1g5saho: line 1: aws: command not found
[2018-06-19 13:45:54,317] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:45:54,316] {bash_operator.py:105} INFO - Command exited with return code 127
[2018-06-19 13:45:54,465] {base_task_runner.py:98} INFO - Subtask: Traceback (most recent call last):
[2018-06-19 13:45:54,466] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/bin/airflow", line 27, in <module>
[2018-06-19 13:45:54,467] {base_task_runner.py:98} INFO - Subtask:     args.func(args)
[2018-06-19 13:45:54,469] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/bin/cli.py", line 392, in run
[2018-06-19 13:45:54,469] {base_task_runner.py:98} INFO - Subtask:     pool=args.pool,
[2018-06-19 13:45:54,471] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/utils/db.py", line 50, in wrapper
[2018-06-19 13:45:54,471] {base_task_runner.py:98} INFO - Subtask:     result = func(*args, **kwargs)
[2018-06-19 13:45:54,472] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/models.py", line 1493, in _run_raw_task
[2018-06-19 13:45:54,473] {base_task_runner.py:98} INFO - Subtask:     result = task_copy.execute(context=context)
[2018-06-19 13:45:54,474] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/operators/bash_operator.py", line 109, in execute
[2018-06-19 13:45:54,475] {base_task_runner.py:98} INFO - Subtask:     raise AirflowException("Bash command failed")
[2018-06-19 13:45:54,476] {base_task_runner.py:98} INFO - Subtask: airflow.exceptions.AirflowException: Bash command failed
[2018-06-19 13:55:39,493] {cli.py:374} INFO - Running on host PRINHYLTPDL1167
[2018-06-19 13:55:39,642] {models.py:1197} INFO - Dependencies all met for <TaskInstance: fileTask.task_1 2018-06-17 08:15:00 [queued]>
[2018-06-19 13:55:39,653] {models.py:1197} INFO - Dependencies all met for <TaskInstance: fileTask.task_1 2018-06-17 08:15:00 [queued]>
[2018-06-19 13:55:39,653] {models.py:1407} INFO - 
--------------------------------------------------------------------------------
Starting attempt 1 of 2
--------------------------------------------------------------------------------

[2018-06-19 13:55:39,697] {models.py:1428} INFO - Executing <Task(BashOperator): task_1> on 2018-06-17 08:15:00
[2018-06-19 13:55:39,698] {base_task_runner.py:115} INFO - Running: ['bash', '-c', u'airflow run fileTask task_1 2018-06-17T08:15:00 --job_id 2 --raw -sd DAGS_FOLDER/fileTask.py']
[2018-06-19 13:55:40,015] {base_task_runner.py:98} INFO - Subtask: /usr/local/lib/python2.7/dist-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
[2018-06-19 13:55:40,015] {base_task_runner.py:98} INFO - Subtask:   """)
[2018-06-19 13:55:40,055] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:55:40,054] {driver.py:120} INFO - Generating grammar tables from /usr/lib/python2.7/lib2to3/Grammar.txt
[2018-06-19 13:55:40,081] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:55:40,081] {driver.py:120} INFO - Generating grammar tables from /usr/lib/python2.7/lib2to3/PatternGrammar.txt
[2018-06-19 13:55:40,216] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:55:40,215] {__init__.py:45} INFO - Using executor SequentialExecutor
[2018-06-19 13:55:40,261] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:55:40,261] {models.py:189} INFO - Filling up the DagBag from /home/charanjiths/airflow/dags/fileTask.py
[2018-06-19 13:55:40,381] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:55:40,381] {bash_operator.py:70} INFO - Tmp dir root location: 
[2018-06-19 13:55:40,382] {base_task_runner.py:98} INFO - Subtask:  /tmp
[2018-06-19 13:55:40,383] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:55:40,382] {bash_operator.py:80} INFO - Temporary script location: /tmp/airflowtmp1iRjXv//tmp/airflowtmp1iRjXv/task_19Ivlqe
[2018-06-19 13:55:40,383] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:55:40,382] {bash_operator.py:88} INFO - Running command:  cp Test_File_2018_06_19.txt /home/charanjiths/airflow/ 
[2018-06-19 13:55:40,386] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:55:40,385] {bash_operator.py:97} INFO - Output:
[2018-06-19 13:55:40,389] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:55:40,389] {bash_operator.py:101} INFO - cp: cannot stat 'Test_File_2018_06_19.txt': No such file or directory
[2018-06-19 13:55:40,392] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 13:55:40,391] {bash_operator.py:105} INFO - Command exited with return code 1
[2018-06-19 13:55:40,472] {base_task_runner.py:98} INFO - Subtask: Traceback (most recent call last):
[2018-06-19 13:55:40,472] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/bin/airflow", line 27, in <module>
[2018-06-19 13:55:40,473] {base_task_runner.py:98} INFO - Subtask:     args.func(args)
[2018-06-19 13:55:40,473] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/bin/cli.py", line 392, in run
[2018-06-19 13:55:40,473] {base_task_runner.py:98} INFO - Subtask:     pool=args.pool,
[2018-06-19 13:55:40,473] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/utils/db.py", line 50, in wrapper
[2018-06-19 13:55:40,474] {base_task_runner.py:98} INFO - Subtask:     result = func(*args, **kwargs)
[2018-06-19 13:55:40,474] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/models.py", line 1493, in _run_raw_task
[2018-06-19 13:55:40,474] {base_task_runner.py:98} INFO - Subtask:     result = task_copy.execute(context=context)
[2018-06-19 13:55:40,474] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/operators/bash_operator.py", line 109, in execute
[2018-06-19 13:55:40,475] {base_task_runner.py:98} INFO - Subtask:     raise AirflowException("Bash command failed")
[2018-06-19 13:55:40,475] {base_task_runner.py:98} INFO - Subtask: airflow.exceptions.AirflowException: Bash command failed
[2018-06-19 14:00:38,482] {cli.py:374} INFO - Running on host PRINHYLTPDL1167
[2018-06-19 14:00:38,649] {models.py:1197} INFO - Dependencies all met for <TaskInstance: fileTask.task_1 2018-06-17 08:15:00 [queued]>
[2018-06-19 14:00:38,661] {models.py:1197} INFO - Dependencies all met for <TaskInstance: fileTask.task_1 2018-06-17 08:15:00 [queued]>
[2018-06-19 14:00:38,662] {models.py:1407} INFO - 
--------------------------------------------------------------------------------
Starting attempt 1 of 2
--------------------------------------------------------------------------------

[2018-06-19 14:00:38,705] {models.py:1428} INFO - Executing <Task(BashOperator): task_1> on 2018-06-17 08:15:00
[2018-06-19 14:00:38,705] {base_task_runner.py:115} INFO - Running: ['bash', '-c', u'airflow run fileTask task_1 2018-06-17T08:15:00 --job_id 2 --raw -sd DAGS_FOLDER/fileTask.py']
[2018-06-19 14:00:38,920] {base_task_runner.py:98} INFO - Subtask: /usr/local/lib/python2.7/dist-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
[2018-06-19 14:00:38,921] {base_task_runner.py:98} INFO - Subtask:   """)
[2018-06-19 14:00:38,953] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 14:00:38,953] {driver.py:120} INFO - Generating grammar tables from /usr/lib/python2.7/lib2to3/Grammar.txt
[2018-06-19 14:00:38,975] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 14:00:38,975] {driver.py:120} INFO - Generating grammar tables from /usr/lib/python2.7/lib2to3/PatternGrammar.txt
[2018-06-19 14:00:39,114] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 14:00:39,112] {__init__.py:45} INFO - Using executor SequentialExecutor
[2018-06-19 14:00:39,168] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 14:00:39,167] {models.py:189} INFO - Filling up the DagBag from /home/charanjiths/airflow/dags/fileTask.py
[2018-06-19 14:00:39,318] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 14:00:39,318] {bash_operator.py:70} INFO - Tmp dir root location: 
[2018-06-19 14:00:39,318] {base_task_runner.py:98} INFO - Subtask:  /tmp
[2018-06-19 14:00:39,319] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 14:00:39,319] {bash_operator.py:80} INFO - Temporary script location: /tmp/airflowtmppTgQRp//tmp/airflowtmppTgQRp/task_19gKLtw
[2018-06-19 14:00:39,320] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 14:00:39,319] {bash_operator.py:88} INFO - Running command:  cp Test_File_2018_06_19.txt /home/charanjiths/airflow/ 
[2018-06-19 14:00:39,324] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 14:00:39,324] {bash_operator.py:97} INFO - Output:
[2018-06-19 14:00:39,328] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 14:00:39,327] {bash_operator.py:101} INFO - cp: cannot stat 'Test_File_2018_06_19.txt': No such file or directory
[2018-06-19 14:00:39,331] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 14:00:39,330] {bash_operator.py:105} INFO - Command exited with return code 1
[2018-06-19 14:00:39,492] {base_task_runner.py:98} INFO - Subtask: Traceback (most recent call last):
[2018-06-19 14:00:39,494] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/bin/airflow", line 27, in <module>
[2018-06-19 14:00:39,503] {base_task_runner.py:98} INFO - Subtask:     args.func(args)
[2018-06-19 14:00:39,504] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/bin/cli.py", line 392, in run
[2018-06-19 14:00:39,511] {base_task_runner.py:98} INFO - Subtask:     pool=args.pool,
[2018-06-19 14:00:39,513] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/utils/db.py", line 50, in wrapper
[2018-06-19 14:00:39,514] {base_task_runner.py:98} INFO - Subtask:     result = func(*args, **kwargs)
[2018-06-19 14:00:39,515] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/models.py", line 1493, in _run_raw_task
[2018-06-19 14:00:39,516] {base_task_runner.py:98} INFO - Subtask:     result = task_copy.execute(context=context)
[2018-06-19 14:00:39,517] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/operators/bash_operator.py", line 109, in execute
[2018-06-19 14:00:39,518] {base_task_runner.py:98} INFO - Subtask:     raise AirflowException("Bash command failed")
[2018-06-19 14:00:39,520] {base_task_runner.py:98} INFO - Subtask: airflow.exceptions.AirflowException: Bash command failed
[2018-06-19 15:51:38,414] {cli.py:374} INFO - Running on host PRINHYLTPDL1167
[2018-06-19 15:51:38,507] {models.py:1197} INFO - Dependencies all met for <TaskInstance: fileTask.task_1 2018-06-17 08:15:00 [queued]>
[2018-06-19 15:51:38,524] {models.py:1197} INFO - Dependencies all met for <TaskInstance: fileTask.task_1 2018-06-17 08:15:00 [queued]>
[2018-06-19 15:51:38,524] {models.py:1407} INFO - 
--------------------------------------------------------------------------------
Starting attempt 1 of 2
--------------------------------------------------------------------------------

[2018-06-19 15:51:38,590] {models.py:1428} INFO - Executing <Task(BashOperator): task_1> on 2018-06-17 08:15:00
[2018-06-19 15:51:38,590] {base_task_runner.py:115} INFO - Running: ['bash', '-c', u'airflow run fileTask task_1 2018-06-17T08:15:00 --job_id 2 --raw -sd DAGS_FOLDER/fileTask.py']
[2018-06-19 15:51:38,817] {base_task_runner.py:98} INFO - Subtask: /usr/local/lib/python2.7/dist-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
[2018-06-19 15:51:38,817] {base_task_runner.py:98} INFO - Subtask:   """)
[2018-06-19 15:51:38,851] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 15:51:38,850] {driver.py:120} INFO - Generating grammar tables from /usr/lib/python2.7/lib2to3/Grammar.txt
[2018-06-19 15:51:38,871] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 15:51:38,870] {driver.py:120} INFO - Generating grammar tables from /usr/lib/python2.7/lib2to3/PatternGrammar.txt
[2018-06-19 15:51:38,979] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 15:51:38,978] {__init__.py:45} INFO - Using executor SequentialExecutor
[2018-06-19 15:51:39,018] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 15:51:39,017] {models.py:189} INFO - Filling up the DagBag from /home/charanjiths/airflow/dags/fileTask.py
[2018-06-19 15:51:39,124] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 15:51:39,124] {bash_operator.py:70} INFO - Tmp dir root location: 
[2018-06-19 15:51:39,124] {base_task_runner.py:98} INFO - Subtask:  /tmp
[2018-06-19 15:51:39,125] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 15:51:39,125] {bash_operator.py:80} INFO - Temporary script location: /tmp/airflowtmp72jcBG//tmp/airflowtmp72jcBG/task_1xjHLtN
[2018-06-19 15:51:39,126] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 15:51:39,125] {bash_operator.py:88} INFO - Running command:  cp Test_File_2018_06_19.txt /home/charanjiths/airflow/ 
[2018-06-19 15:51:39,128] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 15:51:39,128] {bash_operator.py:97} INFO - Output:
[2018-06-19 15:51:39,133] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 15:51:39,133] {bash_operator.py:101} INFO - cp: cannot stat 'Test_File_2018_06_19.txt': No such file or directory
[2018-06-19 15:51:39,135] {base_task_runner.py:98} INFO - Subtask: [2018-06-19 15:51:39,134] {bash_operator.py:105} INFO - Command exited with return code 1
[2018-06-19 15:51:39,199] {base_task_runner.py:98} INFO - Subtask: Traceback (most recent call last):
[2018-06-19 15:51:39,199] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/bin/airflow", line 27, in <module>
[2018-06-19 15:51:39,199] {base_task_runner.py:98} INFO - Subtask:     args.func(args)
[2018-06-19 15:51:39,199] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/bin/cli.py", line 392, in run
[2018-06-19 15:51:39,200] {base_task_runner.py:98} INFO - Subtask:     pool=args.pool,
[2018-06-19 15:51:39,200] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/utils/db.py", line 50, in wrapper
[2018-06-19 15:51:39,200] {base_task_runner.py:98} INFO - Subtask:     result = func(*args, **kwargs)
[2018-06-19 15:51:39,200] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/models.py", line 1493, in _run_raw_task
[2018-06-19 15:51:39,200] {base_task_runner.py:98} INFO - Subtask:     result = task_copy.execute(context=context)
[2018-06-19 15:51:39,201] {base_task_runner.py:98} INFO - Subtask:   File "/home/charanjiths/.local/lib/python2.7/site-packages/airflow/operators/bash_operator.py", line 109, in execute
[2018-06-19 15:51:39,201] {base_task_runner.py:98} INFO - Subtask:     raise AirflowException("Bash command failed")
[2018-06-19 15:51:39,201] {base_task_runner.py:98} INFO - Subtask: airflow.exceptions.AirflowException: Bash command failed
